docs/cloud/sso.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-title: Single Sign On (SSO) for lakeFS Cloud
+title: Single Sign On (SSO)
```

Comment:
this was a comment by @rmoff to distinguish between SSO for Cloud and Enterprise (which are in two different pages), why do you remove it?

Reply:
Removed it because it appears under the menu item for lakeFS Cloud. Same for the "setting up lakeFS Cloud on Azure" one.